pages/common/pwsh.md

```diff
@@ -0,0 +1,20 @@
+# pwsh
+
+> PowerShell Core is a cross-platform automation and configuration tool/framework.
+> More information: <https://docs.microsoft.com/en-us/powershell/>.
+
+- Start an instance of PowerShell:
+
+`pwsh`
+
+- Run the specified script:
+
+`pwsh -File {{path/to/file.ps1}}`
+
+- Set the execution policy for the current session:
+
+`pwsh -ExecutionPolicy {{AllSigned|Bypass|Default|RemoteSigned|Restricted|Undefined|Unrestricted}}`
+
+- Run the specified command as though it was typed into PowerShell:
```

Comment:
In almost all shell pages, we use `Execute a command and then exit`. Do you think this would fit here as well?

Reply:
Sounds fair to me, I'll update it.